hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/handlers/InventoryHandlers.java

```diff
@@ -64,6 +70,45 @@
      */
 
     @GET
+    @Path("/")
```

Comment:
To choose between this json and html output, client should adjust the "accept" header, I think.
If there is anything else it could be nice to have a comment here, I will add it into the rest documentation later.

Reply:
Yes, it's how it works. However there's warning messages in the logs that are bugging me :(
We have the same in hawkular metrics. IIRC it says something like "Two methods found for endpoint, selecting one". Which seems stupid because there's the content negotiation. I don't know if we can get rid of them (without changing the endpoint)

Reply:
Yes, I think is something in the RESTEasy engine.
To get rid a single endpoint can be used, adjusting the output manually checking the accept header.
Some extra code but that would get rid of warnings in log, I think.

Reply:
We should ask the RE folks first. I don't think doing manual parsing is good here, as the Accept header could contain more complex expressions and with weights like application/json;w=0.3,text/html;w=0.7 (not the exact syntax), where despite a/j being listed 1st in reality text/html is preferred.

Reply:
I'm reading about / trying the “quality from server” factor in jax-rs. It may fix the problem.